### PR TITLE
Update common_rules dob in NISRA

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
A simple change from 1 July 2021 to 1 July 2020

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### Northern Ireland

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-nisra-dob/schemas/en/census_household_gb_nir.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-NIR&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/update-nisra-dob/schemas/en/census_individual_gb_nir.json)

### CODE REVIEW

File: source/jsonnet/northern-ireland/lib/common_rules.libsonnet [hunk 1]
```diff
@@ -63,7 +63,7 @@
     id: 'date-of-birth-answer',
     condition: 'greater than',
     date_comparison: {
-      value: '2021-07-01',
+      value: '2020-07-01',
```
Comment (MebinAbraham) [inline source/jsonnet/northern-ireland/lib/common_rules.libsonnet:66]: Does this not mean. the over5 also needs changing.
Current:
```js
schoolYear5OrOver: {
    id: 'date-of-birth-answer',
    condition: 'less than or equal to',
    date_comparison: {
      value: '2021-07-01',
      offset_by: {
        years: -5,
      },
    },
  },
```
5 or Over looks like `50` -_-

Comment (petechd) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet:66]: Screenshot 2020-07-08 at 11 43 27

Comment (ajmaddaford) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet:66]: I mean suggestions for something better would help, maybe `schoolYear5AndOver`?

Comment (MebinAbraham) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet:66]: That works 👍

Comment (LJBabbage) [reply source/jsonnet/northern-ireland/lib/common_rules.libsonnet:66]: schoolYear5AndOver works for me, as it highlights the double condition. I read it as 50 and over and didn't go near it

**Pull request merged**